app/account-service.js

```diff
@@ -0,0 +1,81 @@
+
+
+function AccountService() {
+
+  var ERROR_MESSAGE = "ERROR";
+  var message="ERROR";
+  this.members = [];
+
+
+
+ }
+
+AccountService.prototype.validate = function (mail,password) {
+/*  for (i = 0;i < this.members.length;i++){
+    if ( mail === this.members[i].email  &&
+         password === this.members[i].password) {
+      window.location = "index.html";
+
+      return  "OK";
+    }
+
+  }*/
+  var control = function(a) {
+    if ( mail === a.email  &&
+    password === a.password) {
+      return a;
+    }
+  };
+
+  var  con = this.members.filter (control);
+
+  con[0].user_login = true;
```

Comment:
check if member is not udefined before setting values on it or doing anything else